js_hw_4/task_03.js

```diff
@@ -0,0 +1,63 @@
+// 5*. Напишите рекурсивную функцию, которая принимает на вход число и складывает его цифры. 
+//   Если сумма получилась больше 9 - снова сложите цифры. И так пока, сумма не станет меньше либо равной 9. 
+//   После окончания сложений возвращает полученное число. Например при подаче числа 19 (1+9=10>9, потому 1+0=1) выводится 1
+
+function sumOfDigits(num) {
+    if (!+num){
+        return "The argument must be a number!";
+    } else {
+        const numStr = Math.abs(num).toString();
+        const ArrOfDigits = numStr.split('');
+        let sum = 0;
+        for (digit of ArrOfDigits) {
```

Comment:
```suggestion
        const arrOfDigits = numStr.split('');
        let sum = 0;
        for (digit of arrOfDigits) {
```
only Classes should be named started with the capital letter

Reply:
Will be fixed and delivered in the next commit with next homework